### PR TITLE
[feature1] 1st feature was implemented

### PR DESCRIPTION
**Pull request by renchen-developer**
Following changes where made:
- new-feature.txt
- feature1 folder
    - one-more-file.txt

### CODE REVIEW

**Comment by renchen-developer** [comment]
Not bad. May be add one more file to feature1 folder?

File: feature1/one-more-file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+one mroe file is created in feature1
+
```
Comment (renchen-developer) [inline feature1/one-more-file.txt:2]: good to have a line break here

Comment (renchen-developer) [reply feature1/one-more-file.txt:2]: thanks for comments

**Comment by iixviiiivivi** [comment]
good job

**Pull request merged**